widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Chat.kt

```diff
@@ -13,24 +15,97 @@ internal fun ChatDefaultTheme(pallet: ColorPallet): ChatTheme =
     ChatTheme(
         background = LayerTheme(fill = pallet.backgroundColorTheme),
         header = ChatHeader(pallet),
-        operatorMessage = null,
-        visitorMessage = null,
-        connect = null,
+        operatorMessage = ChatOperatorMessage(pallet),
+        visitorMessage = ChatVisitorMessage(pallet),
+        connect = ChatEngagementStates(pallet),
         input = null,
-        responseCard = null,
-        audioUpgrade = null,
-        videoUpgrade = null,
+        responseCard = ChatResponseCardTheme(pallet),
+        audioUpgrade = UpgradeDefaultTheme(pallet),
+        videoUpgrade = UpgradeDefaultTheme(pallet),
         bubble = null,
         attachmentsPopup = null,
         unreadIndicator = null,
         typingIndicator = null
     )
 
+/**
+ * Default theme for Audio and Video upgrade card
+ */
+private fun UpgradeDefaultTheme(pallet: ColorPallet) = pallet.run {
+    composeIfAtLeastOneNotNull(
+        baseDarkColorTheme,
+        baseNormalColorTheme,
+        primaryColorTheme,
+        baseLightColorTheme,
+        baseShadeColorTheme
+    ) {
+        UpgradeTheme(
+            text = TextTheme(textColor = baseDarkColorTheme),
+            description = TextTheme(textColor = baseNormalColorTheme),
+            iconColor = primaryColorTheme,
+            background = LayerTheme(
+                fill = baseLightColorTheme,
+                stroke = baseShadeColorTheme?.primaryColor
+            )
+        )
+    }
+}
```

Comment:
I would suggest naming it "MediaUpgradeDefaultTheme" and "MediaUpgradeTheme" because just "Upgrade..." is confusing to me. I had to think before I understood what it is. 🙂